src/commcare_cloud/ansible/roles/cloudwatch_logs/tasks/main.yml

```diff
@@ -1,26 +1,18 @@
+---
+
 - name: Configure and start cloudwatch agent
   block:
-  - name: install collectd pre-reqiusities for cloudwatch logs
-    apt:
-      pkg:
-      - collectd
-      update_cache: yes
-      state: present
-  - name: download the amazon cloudwatch agent deb file
-    get_url:
-      url: https://s3.amazonaws.com/amazoncloudwatch-agent/debian/amd64/latest/amazon-cloudwatch-agent.deb
-      dest: /opt/
-      mode: '0666'
-    become: true
-  - name: install cloudwatch deb file
-    apt:
-      deb: "/opt/amazon-cloudwatch-agent.deb"
-      state: present
-    become: true
+    
```

Comment:
@prem-fissionhq I removed your changes here from https://github.com/dimagi/commcare-cloud/pull/4507 because this is a more secure way of doing it. I also had issues with collectd hanging when applying this to india, are you sure collectd is needed?  I can add it back if it's needed.

Reply:
Thanks @knguyen142 for suggestions. Will cross check and get back to you.

Reply:
Not mandatory @knguyen142. We generally use `collectd` software to send the metrics to the CloudWatch agent that's why I have installed it. Please go ahead and remove it. thanks.

Reply:
Ah will we be missing something if we don't have that? We can definitely re-add, I was getting some errors so I removed it, but that might be a one off.

Reply:
I think it's for people who use cloudwatch the way we use datadog. I think there's no need to install collectd right now is my understanding

Reply:
Yes, Correct Danny.